rangeserver/src/cache/memtabledb.rs

```diff
@@ -69,31 +71,31 @@ pub struct MemTable {
 
 #[derive(Debug)]
 pub struct MemTableDB {
```

Comment:
This file could use a one-paragraph top level comment about the design of memtabledb covering the multiple buffer idea and the data structures.